synthetics/resource_test_impl.go

```diff
@@ -196,6 +186,50 @@ func optionalPatchTestFields(test *synthetics.V202101beta1Test) []string {
 		fields = append(fields, "test.settings.url.target")
 	}
 
+	if test.Settings.HasServers() && len(*test.Settings.Servers) != 0 {
```

Comment:
```suggestion
	if test.Settings.HasServers() {
```

I think that user should have a possibility to update server list from non-zero elements to zero elements, i.e. delete elements.

Reply:
Done